<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
alternate-background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Generate a Standard Strength Password</string>
    </property>
    <widget class="QLineEdit" name="lineEdit1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>731</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="standardgenerate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>751</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Generate a Extra Strenght Password</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>731</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="standardgenerate2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>751</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="title">
     <string>Generate a Super Stronge Password</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>731</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="standardgenerate3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
